front/lib/resources/data_source_view_resource.ts

```diff
@@ -32,6 +33,20 @@ import {
 import type { ResourceFindOptions } from "@app/lib/resources/types";
 import type { VaultResource } from "@app/lib/resources/vault_resource";
 
+const getDataSourceCategory = (
```

Comment:
Why not using the one from here?

https://github.com/dust-tt/dust/blob/main/front/lib/api/vaults.ts

Reply:
cycle dependency.
If you look closely, you'll notice I removed it.